elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -0,0 +1,115 @@
+
+ul.connectedSortable {
+  /* This style applies to the MCQ options table, on the left hand side*/
+  border: 2px solid #A9A9A9; /* High visibility borders for accessibility */
+  border-radius: 4px;
+  background-color: #efefff; /* Pale Blue */
+
+  width: 100%;
+
+  margin: 0 40px 0 10px;
+
+  list-style-type: none;
+  padding: 5px 0 20px 0;
+  float: left;
+}
+
+div.right {
+  /* This style applies to the DIV that holds each dropzone column */
+  /* This particular style renders the dropzones in a single row horizontally */ 
+  width: 46%;
+  margin-right: 20px;
+}
+
+div.bottom {
+  /* This style applies to the DIV that holds each dropzone column */
+  /* This particular style renders the dropzones in a single column vertically */ 
+  width: 100%;
+  margin-right: 20px;
+  margin-bottom: 20px;
+}
+
+
+ul.dropzone {
+  /* This style applies to answer dropzone */
+  background-color: #FFA; /* Yellow */
+  min-height: 30px;
+}
+
+
+ul.connectedSortable li, ul.dropzone li{
+  /* This style applies to the answer draggable tiles themselves */
+  background-color: #EFEFEF; /* Grey */
+  border: 2px solid #A9A9A9;
+  border-radius: 4px;
+  margin: 0 10px 10px 10px;
+  padding: 5px;
+  font-size: 1.2em;
+
+  display: block;
+
+  /* Show overflow text in the answer draggable tiles */
+  white-space: normal;
+  height: auto;
+  max-height: none;
+  overflow: visible;
+}
+
+ul.connectedSortable li:hover, ul.dropzone li:hover{
+  cursor: move; /* For UX: The answer tiles shows a 'grab' cursor on mouse over */
+}
+
+
+li.info {
+  /* This styling applies to the instruction text for the dropzones */
+  background-color: transparent !important;
+  border: none !important;
+  cursor: auto !important;
+}
+
+li.ui-state-highlight { 
+  /*This styling applies to the placeholder answer tile that appears when the user is dragging */
+  height: 2em !important; 
+  line-height: 1.5em !important;
+  border: none !important;
+  color: #efefff !important; 
+  background-color: #efefff !important; 
+}
+
+code.user-output {
+  border: 2px solid #A9A9A9; /* High visibility borders for accessibility */
+  margin-right: 4px;
+}
+
+p {
```

Comment:
Please don't ever modify the style of HTML elements without a class selector, as it will globally override the core PL Bootstrap style. If there are `<p>` elements that need this bottom margin, please add a class to them like `pl-order-blocks-set-margin-bottom` (or use a more semantically meaningful name if possible).

Reply:
I found that I added that as a horrible hack because it was hard to get the spacing between sub-questions correct. I learnt that using `<hr>` is much better suited for this purpose, and I removed this styling because it's problematic as you brought up.